GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/contentpage.config

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ContentType Key="4ddb4d6f-6a51-4dfb-a896-9c0e9ac17b96" Alias="contentPage" Level="1">
+  <Info>
+    <Name>Content Page</Name>
+    <Icon>icon-article color-black</Icon>
+    <Thumbnail>folder.png</Thumbnail>
+    <Description></Description>
+    <AllowAtRoot>False</AllowAtRoot>
+    <IsListView>False</IsListView>
+    <Variations>Nothing</Variations>
+    <IsElement>false</IsElement>
+    <HistoryCleanup>
+      <PreventCleanup>False</PreventCleanup>
+      <KeepAllVersionsNewerThanDays></KeepAllVersionsNewerThanDays>
+      <KeepLatestVersionPerDayForDays></KeepLatestVersionPerDayForDays>
+    </HistoryCleanup>
+    <Compositions>
+      <Composition Key="e8386da2-bf9c-43e2-ac0e-02f33f6b6c2d">govukGridColumn</Composition>
```

Comment:
Our example pages add a `blocks` property with a block list or block grid directly on the document type, they don't add this property via a composition. As it happens the document type you chose is not meant to be a composition, and it has the wrong block list data type on it. You should be using 'GOV.UK Block list' or 'GOV.UK Block grid' for an example page.

Reply:
done